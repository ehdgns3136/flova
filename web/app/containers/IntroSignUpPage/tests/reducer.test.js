
import { fromJS } from 'immutable';
import introSignUpPageReducer from '../reducer';

describe('introSignUpPageReducer', () => {
  it('returns the initial state', () => {
    expect(introSignUpPageReducer(undefined, {})).toMatchSnapshot();
  });
});
