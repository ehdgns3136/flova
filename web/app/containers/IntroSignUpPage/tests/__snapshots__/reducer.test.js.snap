exports[`introSignUpPageReducer returns the initial state 1`] = `
Object {
  "isLoading": false,
  "signUpErrorMessage": null,
  "signUpStep": "TermsForm",
  "topicList": Array [],
}
`;
